scripts/rename.js

```diff
@@ -0,0 +1,22 @@
+const {Project} = require("ts-morph")
```

Comment:
Did this sneak in as part of your file renaming PR?

Reply:
Will delete.